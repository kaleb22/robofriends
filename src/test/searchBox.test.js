import { shallow} from 'enzyme'
import React from 'react'
import SearchBox from '../components/SearchBox'

it('renders SearchBox component', () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
})