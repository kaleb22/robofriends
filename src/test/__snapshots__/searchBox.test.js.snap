// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SearchBox component 1`] = `ShallowWrapper {}`;
